packages/@aws-cdk/cloud-assembly-schema/lib/cloud-assembly/artifact-schema.ts

```diff
@@ -223,11 +223,62 @@ export interface NestedCloudAssemblyProperties {
   readonly displayName?: string;
 }
 
+/**
+ * Artifact properties for a feature flag report
+ *
+ * A feature flag report is small enough that all the properties can be inlined
+ * here, and doesn't need an additional file.
+ */
+export interface FeatureFlagReportProperties {
+  /**
+   * The library that this feature flag report applies to.
+   */
+  readonly libraryName: string;
```

Comment:
not sure I like this name. we could use jsii terminology or something more generic like `source`